grails-app/assets/javascripts/streama/streama.run.js

```diff
@@ -1,24 +1,14 @@
-angular.module('streama').run(function ($window, $rootScope, $state, localStorageService, apiService, modalService, userService) {
+angular.module('streama').run(function ($window, $rootScope, $state, localStorageService, apiService, modalService, userService, profileService) {
 	apiService.currentUser().success(function (data) {
 		userService.setCurrentUser(data);
 	});
 
-  apiService.profile.getUserProfiles().success(function(data) {
-    $rootScope.usersProfiles = data;
-      if(!localStorageService.get('currentProfile')){
-        localStorageService.set('currentProfile', data[0]);
-        $rootScope.currentProfile = data[0];
-      }
-    }
-  ).error(function (data) {
-    console.warn(data.message);
-  });
-
-  $rootScope.setCurrentSubProfile = function(profile) {
-    $rootScope.currentProfile = profile;
-    localStorageService.set('currentProfile', profile)
-    $state.go('dash',{},{reload:true});
-  };
+   profileService.getUserProfiles().success(
+    function(data) {
+      $rootScope.usersProfiles = data;
```

Comment:
From what I can see this is only used in the header and (correct me if I'm wrong) it seems the header uses the dash-ctrl, if so we should assign it to a scope variable on the dash-ctrl instead of having it on the rootScope. 

Optimally I'd say this variable should exist in the profile-service and accessed in the relevant controllers instead of having it on the rootScope. Maybe we should talk about this.